core/db/default/spree/permission_sets.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+Spree::PermissionSets::Base.subclasses.each do |permission|
+  Spree::PermissionSet.create(name: permission.to_s.split('PermissionSets::').last, set: permission.to_s)
```

Comment:
Ah, here we could map a potential static `permission_type` class method to a `permission_type` column, right?